app/logical/sources/strategies/twitter.rb

```diff
@@ -47,49 +47,59 @@ def page_url
     end
 
     def profile_url
+      return "" if api_response.blank?
```

Comment:
This should probably go on line 55. This strategy will sometimes be used by ArtistUrl to normalize profile urls, so sometimes `url` will be the profile url.

Reply:
Should be fixed in 6c94047. The artist url test for this now passes.